docker-compose.yml

```diff
@@ -0,0 +1,10 @@
+version: '3'
+services:
+  api:
+    build: ./api
+    ports: ["3001:3001"]
+    volumes: ['./api:/app']
+  client:
+    build: ./client
+    ports: ["3000:3000"]
+    volumes: ['./client:/app']
```

Comment:
@nazar  I think you should add a [depends_on](https://docs.docker.com/compose/compose-file/#depends_on) attribute here so the api is always started before the client.